app/views/building_map/index.html.erb

```diff
@@ -43,14 +43,18 @@
 
 <datalist id="startOptions">
   <option><%= BuildingMapController::YOUR_LOCATION_MAGIC_STRING %></option>
-  <% Places::DESTINATIONS.each do |dest, coords| %>
+  <option><%= BuildingMapController::PIN_1_MAGIC_STRING %></option>
+  <option><%= BuildingMapController::PIN_2_MAGIC_STRING %></option>
+  <%= Places::DESTINATIONS.each do |dest, coords| %>
```

Comment:
```suggestion
  <% Places::DESTINATIONS.each do |dest, coords| %>
```
That is, how it was before. Are you sure, that the `=` is correct?
Just want to make sure.

Reply:
There should be no `=`, because this prints the first expression in the following code to the website. In this case, `Places::DESTINATIONS` would be dumped into the HTML (although our CSS makes it invisible). That was also why I purposefully removed the `=` in #137.

Reply:
And don't forget to also change this back for the other datalist.